.github/workflows/checkIssueInfo.yaml

```diff
@@ -0,0 +1,29 @@
+name: Check for missing information
+on: 
+  issues:
+    types: [opened]
+jobs: 
+  checkIssueInformation:
+    name: Check for missing information
+    runs-on: ubuntu-latest
+    env:
+      issue_incomplete: 'False'
```

Comment:
hmm what does this mean?

Reply:
That is an environment variable that is loaded with the response from the script that checks the contents of the issue in the first step of the job. I initialize it like that because it has to have a value from the start for it to work, but it will be overridden when the script runs. This is then used in the second step to check if the label and comment should be added. I can change it to something like 'undefined' if that's better

Reply:
The current value is fine, can you add these as comments

Reply:
Done